docs/ReleaseNotes.md

```diff
@@ -18,6 +18,7 @@ In Acuminator 3.1.2, the following enhancements have been implemented:
    - The diagnostic now supports the `inheritdoc` tag for DACs, DAC extensions, and their properties. The diagnostic also checks that mapped DAC field properties of projection DACs have the `inheritdoc` tag with the `cref` attribute pointing to the corresponding DAC field property of the original DAC. 
    - A new code fix can generate the `inheritdoc` tag for a mapped DAC property of a projection DAC.
    - Code fixes of the diagnostic can now generate documentation correctly even for a badly formatted code and remove incorrect documentation tags in case of mapped projection DAC field properties. The code fixes also support doc tags declared as empty XML elements such as `<summary/>`.
+   - The Acuminator "Suppress with a local comment" code fix now supports Visual Studio "Fix All" functionality for PX1007 diagnostic. The diagnostic can be suppressed for a whole DAC. This could be useful when you do not plan to write the documentation immediately and want to leave it to the documentation team.
```

Comment:
@EAndrosova  please check this addition to release note with the description of the change